development/architecture/migration-guide/horizontal-migration/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: How to migrate Back Office pages to Symfony with Horizontal migration
+menuTitle: Horizontal migration guide
+chapter: true
+weight: 40
+summary: "The guide we use to migrate pages to Symfony"
+---
+
+# How to migrate Back Office pages to Symfony
+
+Migrating a legacy page in PrestaShop requires working on three parts of the application: templates, forms and controllers which contain the business logic.
```

Comment:
> which contain the business logic.

IMO that last part doesn't seem to add much value. (it sounds as if the main responsibility of all those parts is to contain business logic, whereas in theory neither of them should and there is no purpose to explain their responsibility here anyway).

Also maybe its worth mentioning the main abstract idea of this migration (at least at this stage) - that all those parts will have to be wrapped into dedicated adapters which will still use legacy services (so that way it would be clearer why you only mention presentation layer (forms, controllers, templates) and nothing related to domain (like entities, value objects, domain services etc..)